compliance/azure/ahub_manual/README.md

```diff
@@ -21,6 +21,12 @@ Each license is good for one VM with up to 16 cores or two VM's with up to 8 cor
 - policy_designer
 - policy_manager
 - policy_publisher
+- credential_viewer
```

Comment:
for consistency please change to Cloud Management - credential_viewer

Reply:
Changed to Cloud Management - credential_viewer